tools/otci/tests/test_otci.py

```diff
@@ -402,7 +402,14 @@ def _test_otci_multi_nodes(self, leader, commissioner, child1, child2):
         self.assertEqual('router', commissioner.get_state())
 
         for dst_ip in leader.get_ipaddrs():
-            commissioner.ping(dst_ip, size=10, count=1, interval=2, hoplimit=3)
+            statistics = commissioner.ping(dst_ip, size=10, count=10, interval=2, hoplimit=3)
+            self.assertTrue('transmitted_packets' in statistics)
+            self.assertTrue('received_packets' in statistics)
+            self.assertTrue('packet_loss' in statistics)
+            self.assertTrue('round_trip_time' in statistics)
+            self.assertTrue('min' in statistics['round_trip_time'])
+            self.assertTrue('avg' in statistics['round_trip_time'])
+            self.assertTrue('max' in statistics['round_trip_time'])
```

Comment:
Can we verify whether the statistics are correct?

Reply:
I don't have much idea about how to do this verification, especially for the RTTs.

Reply:
Per offline discussion, added some basic validations.